fedimint-client/src/db.rs

```diff
@@ -1,23 +1,18 @@
-use std::io::{Error, Read, Write};
-use std::marker::PhantomData;
-
 use fedimint_core::api::{ApiVersionSet, InviteCode};
 use fedimint_core::config::{ClientConfig, FederationId};
 use fedimint_core::core::OperationId;
-use fedimint_core::encoding::{Decodable, DecodeError, Encodable};
-use fedimint_core::module::registry::ModuleDecoderRegistry;
+use fedimint_core::encoding::{Decodable, Encodable};
 use fedimint_core::{impl_db_lookup, impl_db_record};
 use serde::Serialize;
 use strum_macros::EnumIter;
 
 use crate::oplog::OperationLogEntry;
-use crate::secret::RootSecretStrategy;
-use crate::ClientSecret;
 
 #[repr(u8)]
 #[derive(Clone, EnumIter, Debug)]
 pub enum DbKeyPrefix {
-    ClientSecret = 0x29,
+    EncodedClientSecret = 0x28,
+    ClientSecret = 0x29, // Unused
```

Comment:
**question:** If unused, why not delete it? If we want to make sure we won't re-use it, maybe we should change the comment `// tombstone - used in the past` or something.

Reply:
I wasn't sure if there's a need to ensure that it doesn't get re-used. I can just remove it.

Reply:
I'm also not sure. Probably safer to leave with a good comment and remove in the future.

Reply:
Why not keep the same key? From a first glance you don't seem to change the encoding?